ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -29,6 +29,14 @@ const Allele = styled.span`
   font-style: ${(props) => { return props.isRef ? 'inherit' : 'italic' }};
 `
 
+const PAR_REGIONS = {
+  37: [[60001, 2699521], [154931044, 155260561]],
```

Comment:
we get variant calls on both X and Y so would include the Y intervals also

Reply:
but there us a check for `variant.chrom === 'X'` so when would that happen?

Reply:
It would have to be something like
```
const PAR_REGIONS = {
  37: {
    'X': [[60001, 2699521], [154931044, 155260561]],
    'Y': [..., ],
   },
  38: {
    'X': [..., ],
    'Y': [..., ],
  }

const isHemiVariant = variant =>
  (variant.chrom === 'X' ||   variant.chrom === 'Y')  && PAR_REGIONS[variant.genomeVersion][variant.chrom].every(region => !(region[0] < variant.pos < region[1]))
```

Reply:
Also, it might be better to make the 'M' check part of isHemi:
```
const isHemiVariant = (variant, individual) => (individual.sex === 'M' && (variant.chrom === 'X' ||   variant.chrom === 'Y') ...)```